src/hooks/mongodb3x.test.js

```diff
@@ -1,31 +1,33 @@
 import { HandlerInputsBuilder } from '../../testUtils/handlerInputsBuilder';
-import {
-  getMockedMongoClient,
-  promisifyMongoFunc,
-  wrapMongoCollection,
-} from '../../testUtils/mongoMocker';
+import { getMockedMongoClient, wrapMongoCollection } from '../../testUtils/mongo3xMocker';
 import { MongoSpanBuilder } from '../../testUtils/mongoSpanBuilder';
 import { SpansContainer, TracerGlobals } from '../globals';
 import { hookMongoDb } from './mongodb';
 
 const DUMMY_URL = 'mongodb://localhost:27017/myproject';
 
 describe('mongodb', () => {
+  let connection;
```

Comment:
Why extract this to global, it's going to create flakiness when running parallel

Reply:
that's a fair point, reverting